frontend/src/app/Jira/Jira.tsx

```diff
@@ -229,61 +175,354 @@ export const Jira = () => {
-                                            <div style={{ cursor: "pointer" }} aria-expanded={isExpanded} onClick={event => showJiras(JIRA_ALL)}>
-                                                <Title headingLevel="h2" size={TitleSizes['lg']}>All</Title>
-                                                {allIssueChart()}
-                                            </div>
-                                        </CardBody>
-                                    </GridItem>
-                                    <GridItem style={{ cursor: "pointer" }} aria-expanded={isExpanded} onClick={event => showJiras(JIRA_BLOCKER)}>
-                                        <CardBody style={{ display: "flex", alignItems: "center", justifyContent: "center", minHeight: "200px", margin: "auto 5px", textAlign: "center" }}>
-                                            <div>
-                                                <Title headingLevel="h2" size={TitleSizes['lg']}>Blocker</Title>
-                                                {issueChart(JIRA_BLOCKER)}
-                                            </div>
-                                        </CardBody>
-                                    </GridItem>
-                                    <GridItem style={{ cursor: "pointer" }} aria-expanded={isExpanded} onClick={event => showJiras(JIRA_CRITICAL)}>
-                                        <CardBody style={{ display: "flex", alignItems: "center", justifyContent: "center", minHeight: "200px", margin: "auto 5px", textAlign: "center" }}>
-                                            <div>
-                                                <Title headingLevel="h2" size={TitleSizes['lg']}>Critical</Title>
-                                                {issueChart(JIRA_CRITICAL)}
-                                            </div>
-                                        </CardBody>
+            <PageSection>
+                <React.Fragment>
+                <Grid hasGutter>
+                    <GridItem order={{default: "2"}}>
+                        <Grid hasGutter sm={6} md={6} lg={6} xl={6}>
+                            <GridItem order={{default: "1"}}>
+                                <Card style={{textAlign: 'center'}}>
+                                    <CardTitle style={{textAlign: 'center'}}>Average Resolution Time (for past 12 months)</CardTitle>
+                                    <CardBody>
+                                        <Title headingLevel='h1' size="2xl">
+                                            { apiDataCache[selected] &&
+                                            <span>
+                                                <span>{ parseFloat(apiDataCache[selected].resolved.total).toFixed(2) || "-"}</span>
+                                                <span style={{paddingLeft: '5px', fontSize: '15px', fontWeight: 'normal'}}>hours</span>
```

Comment:
Maybe converting the `Average Resolution Time` to days would be easier to read. WDTY?

Reply:
Yep, it's good idea. But it depends on @flacatus: he's doing the magic calculus in backend. @flacatus WDYT?

Reply:
for me works to transform in frontend